app/controllers/admin/orders_controller.rb

```diff
@@ -3,7 +3,23 @@
 module Admin
   class OrdersController < AdminController
     def index
-      render :index, locals: { orders: Order.includes(:employee).all.order(:status) }
+      # render index, locals: { orders: Order.includes(:employee).all.order(:status) }
+      @orders = Order.includes(:employee).all.order(:status)
+
+      # respond_to do |format|
+      #  format.html
+      #  format.csv { send_data @orders.to_csv, filename: "orders-#{Date.today}.csv" }
+      # end
+
+      respond_to do |format|
+        format.html
+        # render :template => 'admin/orders/index.html.erb'
+        format.csv do
+          response.headers['Content-Type'] = 'text/csv'
+          response.headers['Content-Disposition'] = "attachment; filename=Orders-#{Time.zone.today}.csv"
+          render template: 'admin/orders/export.csv.erb'
```

Comment:
Calling methods with `.` is deprecated according to rspec
```suggestion
          render template: 'admin/orders/export', handlers: [:erb], formats: [:csv]
```

Reply:
Indeed rubocop reports this entry as incorrect